pyproject.toml

```diff
@@ -25,12 +25,13 @@ backend = [
     "setproctitle>=1.3.6",
     "weave>=0.51.51",
     "tblib>=3.0.0",
-    "semver>=3.0.4",
     "setuptools>=78.1.0",
     "wandb>=0.19.8",
     "polars>=1.26.0",
 ]
 
+skypilot = ["semver>=3.0.4", "skypilot==0.8.0"]
```

Comment:
@Kovbo In our dev-dependencies block, we have this form of skypilot installed:

```
"skypilot[cudo,do,fluidstack,gcp,lambda,paperspace,runpod]==0.8.0"
```

Is it important that we include all of these extras?

Reply:
@corbt We should also get rid of the skypilot dependency in dev-dependencies block.